### DISCUSSION TITLE
夸克网盘加载过慢

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#6487] [Q&A]
请问各位，夸克网盘挂载之后是不是特别慢，我看1080p的视频都一卡一卡的，用aria下载也只有2mb，百度网盘都有20mb

**Comment by helloint** [comment]
我也是

**Comment by helloint** [comment]
我发现用Alist自带的player播放并不慢, 说明token并没问题, 没有限速.
但我用Infuse挂载webdav, 就挺慢的, 打开后有明显的loading等待
